### PR TITLE
Drop commented out / dead code

### PR DESCRIPTION
**Pull request by raharper**

### CODE REVIEW

File: subiquity/models/locale.py [hunk 1]
```diff
@@ -36,7 +36,6 @@ class LocaleModel(object):
         ('fr_FR', 'French'),
         ('de_DE', 'German'),
         ('el_GR', 'Greek, Modern (1453-)'),
-#        ('he_IL', 'Hebrew'), # disabled as it does not render correctly on a vt with default font
```
Comment (mwhudson) [inline subiquity/models/locale.py]: I want to keep this one for when we start using bterm or something.

Comment (raharper) [reply subiquity/models/locale.py]: That's fine, we can add a 'noqa:  after the second hash before your disabled comment.

Comment (mwhudson) [reply subiquity/models/locale.py]: Do you want to do that and then merge this branch then?

**Comment by raharper** [comment]
Updated and force pushed to this branch, after CI says ok, I can merge

**Pull request merged**